client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts

```diff
@@ -271,4 +271,23 @@ export class WildFiresListComponent extends CollectionComponent implements OnCha
     this.doSearch()
   }
 
+  isLocationName() {
+    if (this.locationName && this.locationName !== ""){
```

Comment:
just return the conditional:

`return this.locationName && this.locationName !== ''`

Reply:
return this.locationName && this.locationName !== ''  doesn't really work here.
when this.locationName = '', return ( this.locationName && this.locationName !== '' ) will not return a boolean. 
It is actually return ( '' & false) , and the result is not false

Reply:
Right. You'd have to define the conditional to get rid of the behaviour when using `if (this.locationName)`